test/e2e/search_test.go

```diff
@@ -423,4 +423,15 @@ registries = ['{{.Host}}:{{.Port}}']`
 		Expect(search.ExitCode()).To(Equal(0))
 		Expect(len(search.OutputToStringArray()) > 1).To(BeTrue())
 	})
+
+	It("podman search repository tags", func() {
+		search := podmanTest.Podman([]string{"search", "--list-tags", "--limit", "30", "docker.io/library/alpine"})
+		search.WaitWithDefaultTimeout()
+		Expect(search.ExitCode()).To(Equal(0))
+		Expect(len(search.OutputToStringArray())).To(Equal(31))
+
+		search = podmanTest.Podman([]string{"search", "--filter=is-official", "--list-tags", "docker.io/library/alpine"})
+		search.WaitWithDefaultTimeout()
+		Expect(search.ExitCode()).To(Not(Equal(0)))
+	})
```

Comment:
need to add a search for a registry too with a backslash only, no image name.  For that and for this test, add a test to make sure the returned lines are greater than 1 and perhaps greater than 2 or 3.

Reply:
I see you added the test with lines >2, ty!  Could you also add a test that lists the tags for `docker.io/library/` (no image, just a backslash).  According to the doc, it's allowed:  `podman search registry.fedoraproject.org/` and if it is NOT allowed with the `--list-tags` option, it would be good to test that.

Reply:
Test added. It's not allowed, no tags will be returned for `docker.io/library/`

Reply:
I'm fine with not allowing it, but we really should add a note to the `--list-tags` option in the man page saying that, and preferably raise some kind of error message telling the user that they need to specify an image or at least return an "image not found" kind of message.

Reply:
Hope I'm not being too painful @QiWang19 !

Reply:
the error would be "library/" must be a docker reference". Add to the doc "the search term should be repository name". Would this work?
$ bin/podman search --list-tags docker.io/library/
ERRO[0000] error listing registry tags "docker.io": reference "library/" must be a docker reference

Reply:
OK, first off, I'd still like to see a test for `docker.io/library/` (or some other registry) and then test for the appropriate error message.  I'm finding the man page to be very confusing at the moment, will try to suggest a rewrite.

Reply:
search does not return error, it logs errors to standard logger and returns the 0 entry of the result.

Reply:
Sounds good and it looks like you've added a test for that, TY!